src/userlist/userListActions.js

```diff
@@ -21,3 +20,9 @@ export const sendGetUsers = (auth: Auth) =>
       users: response,
     });
   };
+
+export const sendInitialGetUsers = (auth: Auth) =>
```

Comment:
This is to differentiate from the action and the api call called getUsers. We can use other names, but they should definitely differ.